src/extractor.py

```diff
@@ -0,0 +1,61 @@
+import re
+import time
+
+from tweets import Tweets
+
+def get_text():
+    consumer_key = ""
+    consumer_secret = ""
+    access_key = ""
+    access_secret = ""
+
+    twitter = Tweets(consumer_key, consumer_secret, access_key, access_secret)
+
+    hashtags = set()
+    handles = []
+
+    with open("../resources/twitter_handles.csv") as f:
```

Comment:
[nit] I don't like relative paths in code, I prefer having the methods take them in as params and the driver code supplies them (or they are specified in the Makefile. For example see `driver.go` and how it calls all the other components)